AEPIdentity/Tests/IdentityTests.swift

```diff
@@ -548,7 +545,25 @@ class IdentityTests: XCTestCase {
         mockRuntime.simulateSharedState(extensionName: "com.adobe.module.analytics", event: getUrlVariablesEvent, data: ([IdentityConstants.Analytics.ANALYTICS_ID: "test-aid"], .set))
 
         // verify
-        XCTAssertFalse(identity.readyForEvent(getUrlVariablesEvent)) // booting up
-        XCTAssertTrue(identity.readyForEvent(getUrlVariablesEvent))
+        XCTAssertTrue(identity.readyForEvent(getUrlVariablesEvent)) // fast boot
+    }
+
+    func testReadyForEventGetEcidOnAppInstallWillWaitForConfigurationAndECIDToBeGenerated() {
```

Comment:
Can you verify in some way that the event actually is waiting on the configuration state? Maybe something like:

1. call readyForEvent and verify False
2. verify ECID is not set in Identity Properties
3. set valid Configuration
4. call readyForEvent and verify True
5. verify ECID is set in Identity Properties.